custom/icds_reports/models/views.py

```diff
@@ -193,6 +193,8 @@ class ChildHealthMonthlyView(models.Model):
     current_month_wasting_v2 = models.TextField(blank=True, null=True)
     current_month_stunting_v2_sort = models.IntegerField(blank=True, null=True)
     current_month_wasting_v2_sort = models.IntegerField(blank=True, null=True)
+    contact_phone_number = models.TextField(blank=True, null=True, help_text="aww phone number")
+    phone_number = models.TextField(blank=True, null=True, help_text="parent phone number")
```

Comment:
you added both of these here, but only one to the actual view, does it need both?

Reply:
Adding the 'contact_phone_number' to view model (as it already was in SQL) allows for sorting by it, which is helpful in beneficiary list report as clicking on column header sets sorting by this field.